services/google_maps_service.js

```diff
@@ -0,0 +1,25 @@
+var fetch = require('node-fetch');
+
+class googleMapsService {
```

Comment:
@bplantico

Reply:
If it looks like a Google Geocoding API service, and it talks like a Google Geocoding API service...